Package.swift

```diff
@@ -0,0 +1,8 @@
+import PackageDescription
+
+let package = Package(
+  name: "JWT",
+  dependencies: [
+    .Package(url: "https://github.com/krzyzanowskim/CryptoSwift", versions: Version(0,2,2)..<Version(1,0,0))
```

Comment:
We're using and testing against [CryptoSwift 0.1.1 in the podspec](https://github.com/kylef/JSONWebToken.swift/blob/master/JSONWebToken.podspec#L14), would you be able to update the pod spec to use `~> 0.2.0` so it's the same here?

---

As per semver, with versions less than 0, any minor updates can cause API changes and thus we shouldn't allow versions 0.2.2 to 1.0.0. Can we change this version range to the following so upstream changes won't break JWT:

```swift
Version(0, 2, 2) ..< Version(0, 3, 0)
```